flipperdevices/Flipper-Android-App
# Fix infinite dispatch

**Background**

Remote controls screens dispatching signals even when screen is closed. This PR introduces fix for this behaviour 

**Changes**

- Fix infinite dispatch on infrareds screen
- Move CloseEmulateAppTaskHolder into api
- Fix self-hosted runners on release workflow

**Test plan**

- Open remote-controls setup or remote-control screen
- Press emulate and close screen
- See emulaste work is stopped
